lib/app/initializers/02_log_level.rb

```diff
@@ -0,0 +1,4 @@
+logger = AppConfig.connectors_logger
```

Comment:
needs license header and requires

Reply:
Good call, I wonder why pipeline is green when I don't have license headers :hmm: